zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -83,14 +87,14 @@ def get_bot_converter_response(message, bot_handler):
                     exponent -= exp
                     unit_to = unit_to[len(key):]
 
-            uf_to_std = utils.UNITS.get(unit_from, False)
-            ut_to_std = utils.UNITS.get(unit_to, False)
+            uf_to_std = utils.UNITS.get(unit_from, [])
+            ut_to_std = utils.UNITS.get(unit_to, [])
 
-            if uf_to_std is False:
+            if uf_to_std == []:
                 results.append('`' + unit_from + '` is not a valid unit. ' + utils.QUICK_HELP)
-            if ut_to_std is False:
+            if ut_to_std == []:
                 results.append('`' + unit_to + '` is not a valid unit.' + utils.QUICK_HELP)
-            if uf_to_std is False or ut_to_std is False:
+            if uf_to_std == [] or ut_to_std == []:
```

Comment:
I think its more Pythonic to say `if not uf_to_std` instead of explicitly checking whether it is equal to an empty list. I would recommend using `if not uf_to_std` for all of these. This is again a PEP8 thing, which I highly recommend reading. :) Here's [a link](https://www.python.org/dev/peps/pep-0008/).